File: backend/backend/src/main/java/com/backend/backend/Data/Log.java
package com.backend.backend.Data;

import org.hibernate.annotations.Type;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import java.util.UUID;


@Entity
public class Log {

    @Id
    @GeneratedValue
    @Type(type="uuid-char")
    private UUID id;

    private String event;

    private String date;

    public Log(String event, String date) {
        this.event = event;
        this.date = date;
    }
    public Log() {

    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getEvent() {return this.event;}
    public void  setEvent(String event) {this.event = event;}

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
}

File: ClientApp/app/caffadminapp/src/main/java/com/e/caffadminapp/model/Log.kt
package com.e.caffadminapp.model

class Log {

    private lateinit var event: String

    private lateinit var date: String

    fun getEvent(): String {
        return event
    }

    fun setEvent(event: String) {
        this.event = event
    }

    fun getDate(): String {
        return date
    }

    fun setDate(date: String) {
        this.date = date
    }

    constructor(event: String, date: String) {
        this.event = event
        this.date = date
    }
}